example/android/app/build.gradle

```diff
@@ -100,6 +100,6 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:23.4.0"
-    compile "com.facebook.react:react-native:0.29.0"  // From node_modules
+    compile "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
Correct. [Here's](https://docs.gradle.org/current/userguide/dependency_management.html) the documentation on how it works:

```
Newest: The newest version of the dependency is used. This is Gradle's default strategy, and is often an appropriate choice as long as versions are backwards-compatible.
```

React Native in uses weird and non-standard ways of doing some stuff, this is one of them. We'll try to have them change their minds on this :)